src/gui/gui_console.cpp

```diff
@@ -1486,9 +1486,17 @@ void ui::console_edit::edit_box_enter(sys::state& state, std::string_view s) noe
 	case command_info::type::complete_constructions:
 		command::c_complete_constructions(state, state.local_player_nation);
 		break;
-	case command_info::type::instant_research:
+	case command_info::type::instant_research: {
+		auto has_us = false;
+		for(const auto n : state.cheat_data.instant_research_nations)
+			if(n == state.local_player_nation) {
+				has_us = true;
+				break;
+			}
+		log_to_console(state, parent, !has_us ? "\xA7GEnabled" : "\xA7RDisabled");
```

Comment:
shouldn't this come from the localization csv file?

Reply:
logging to console works a bit different and cant make use of localization csv

Reply:
atleast not w/o discarding colour info

Reply:
yes you can; you can grab the csv entry and convert it to a string and then pass the string to the log to console function

Reply:
the idea is to have COLOR

Reply:
Now that I think of it, it would be more efficient to make use of the checkmark/cross symbols we already have

Reply:
what is wrong with "\xA7G" + to_string(csv_key) ?

Reply:
> what is wrong with "\xA7G" + to_string(csv_key) ?

a checkmark will do then